### PR TITLE
Payout oracles when set_billing is called

### PR DESCRIPTION
**Pull request by aalu1418**

### CODE REVIEW

File: contracts/ocr2/src/integration_tests.rs [hunk 1]
```diff
@@ -801,3 +801,68 @@ fn set_link_token() {
         .unwrap();
     assert_eq!(token_addr, new_link_token);
 }
+
+#[test]
+fn set_billing_payout() {
+    let mut env = setup();
+    let deposit = Decimal::from_str("1000").unwrap().0;
```
Comment (krebernisak) [inline contracts/ocr2/src/integration_tests.rs:808]: Seems unused in the test.

File: contracts/ocr2/src/integration_tests.rs [hunk 2]
```diff
@@ -801,3 +801,68 @@ fn set_link_token() {
...
+    let observation_payment = Uint128::from(5 * GIGA);
+    let reimbursement = Decimal::from_str("0.001871716").unwrap().0;
+
+    // -- set billing
+    // price in uLUNA
+    let recommended_gas_price = Decimal::from_str("0.01133").unwrap();
+    let msg = ExecuteMsg::SetBilling {
+        config: Billing {
+            recommended_gas_price_micro: recommended_gas_price,
+            observation_payment_gjuels: 5,
```
Comment (krebernisak) [inline contracts/ocr2/src/integration_tests.rs:819]: Consider changing to some different/new billing values here.

**Review by krebernisak — APPROVED** [review]

File: contracts/ocr2/src/integration_tests.rs (hunk 1 above)
Comment (aalu1418) [reply contracts/ocr2/src/integration_tests.rs]: resolved the changes during the rebase fixes

File: contracts/ocr2/src/integration_tests.rs (hunk 2 above)
Comment (aalu1418) [reply contracts/ocr2/src/integration_tests.rs:958]: resolved the changes during the rebase fixes

**Review by krebernisak — APPROVED** [review]

**Review by connorwstein — APPROVED** [review]

**Pull request merged**